Reference/Management/Models/Content.md

```diff
@@ -65,7 +64,7 @@ return content.CreatorId;
 
 ### .ContentType
 
-Returns a `ContentType` object representing the DocumentType used by the given `Content`.
+Returns a `ISimlpeContentType` object representing the DocumentType used by the given `Content`.
```

Comment:
Would you mind double checking the return object itself @erikjanwestendorp 
I assume there's a typo here :)

Reply:
@busrasengul As far as I can see it returns ISimlpeContentType. 
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Models/ContentBase.cs#L83

Reply:
ISimpleContentType then but your PR says ISimlpeContentType. So there's a typo 😊 (l and p swapped places)

Reply:
Hahah oke, my bad 🙈. Just fixed it!